### DISCUSSION TITLE
Incorrect casing on user.name means I am not showing as a contributor on repos.

### DISCUSSION THREAD

**Discussion by tjk3o** [community/community#28307] [Other Feature Feedback, Questions, & Ideas]
I have noticed that I am not showing as a contributor on several repos that I work on. The reason is that I had my handle set in uppercase in my local git config, whereas my actual handle is lowercase. Is there a way for me to "merge" the 2 so that I can show up as a contributor for all my previous commits?

**Comment by jfullstackdev** [comment]
do you use the same email ? commits are linked to emails, so if you used another email for that, then add that email on GitHub, not necessarily primary, just add it even unverified
